test/framework/filetools.py

```diff
@@ -1635,36 +1635,17 @@ def test_create_patch_info(self):
         self.assertEqual(ft.create_patch_info('foo.patch'), {'name': 'foo.patch'})
         self.assertEqual(ft.create_patch_info(('foo.patch', 1)), {'name': 'foo.patch', 'level': 1})
         self.assertEqual(ft.create_patch_info(('foo.patch', 'subdir')), {'name': 'foo.patch', 'sourcepath': 'subdir'})
-        self.assertEqual(ft.create_patch_info(('foo.txt', 'subdir')), {'name': 'foo.txt', 'copy': 'subdir'})
```

Comment:
Hmm, don't we still have cases like this, where the file specified in `patches` should just be copied?

Why does this case need to be removed exactly?

Reply:
It does not. I'd misunderstood what we were testing here. I've restored those in f1b58b3